gitlab/__init__.py

```diff
@@ -189,6 +192,8 @@ def set_url(self, url):
         self._url = '%s/api/v3' % url
 
     def _construct_url(self, id_, obj, parameters):
+        if 'next_url' in parameters:
```

Comment:
@gpocentek I'm not sure I understand the purpose of that, given how this function is called. Even if I don't use the generator idea, from @dserodio, it's your project but I'd rather not add in additional unused code paths...